.nais/alerts-dev.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "nais.io/v1"
+kind: "Alert"
+metadata:
+  name: amt-tiltak-alerts
+  namespace: amt
+  labels:
+    team: amt
+spec:
+  receivers:
+    slack:
+      channel: 'team_komet_alerts'
+      prependText: '<!here> | '
+  alerts:
+    - alert: applikasjon nede
```

Comment:
Kunne kanskje vært en fordel med app navn i tittelen. Eller er ikke sikker på hvordan det blir seende ut. "Amt-tiltak er nede" for eksempel

Reply:
Skal vi ta å få testa hvordan det blir seendes ut først?